fabrictestbed_extensions/fablib/resources.py

```diff
@@ -746,6 +764,32 @@ def site_to_dictXXX(self, site):
                 "value": self.get_component_capacity(site_name, "GPU-RTX6000")
                 - self.get_component_available(site_name, "GPU-RTX6000"),
             },
+            "a30_available": {
+                "pretty_name": "A30 Available",
+                "value": self.get_component_available(site_name, "GPU-A30"),
+            },
+            "a30_capacity": {
+                "pretty_name": "A30 Capacity",
+                "value": self.get_component_capacity(site_name, "GPU-A30"),
+            },
+            "a30_allocated": {
+                "pretty_name": "A30 Allocated",
+                "value": self.get_component_capacity(site_name, "GPU-A30")
+                - self.get_component_available(site_name, "GPU-A30"),
+            },
+            "a40_available": {
+                "pretty_name": "A40 Available",
+                "value": self.get_component_available(site_name, "GPU-A40"),
+            },
+            "a40_capacity": {
+                "pretty_name": "A40 Capacity",
+                "value": self.get_component_capacity(site_name, "GPU-A40"),
+            },
+            "a40_allocated": {
+                "pretty_name": "A40 Allocated",
+                "value": self.get_component_capacity(site_name, "GPU-A40")
+                - self.get_component_available(site_name, "GPU-A40"),
```

Comment:
Just curious: why is `"value"` computed differently?  In some cases it is capacity - available, in others it is just capacity. Is this intentional?

Reply:
Never mind, it makes sense on second reading.